common/src/main/java/com/wynntils/screens/wynntilsmenu/WynntilsMenuScreen.java

```diff
@@ -218,6 +218,18 @@ private static void renderPlayerInfo(PoseStack poseStack, int mouseX, int mouseY
         InventoryScreen.renderEntityInInventoryFollowsMouse(
                 poseStack, posX, posY, 30, posX + 45 - mouseX, posY - 40 - mouseY, McUtils.player());
 
+        FontRenderer.getInstance()
```

Comment:
Would it be better to only render this is `Models.Character.getGuild()` isn't blank or is this fine?

Reply:
Yes, there should be a check for if the player is actually in a guild or not, and skip the render if they're not. No point rendering empty strings.